### ISSUE TITLE
Lure of the Wildcat Windurst - Quests interfering with NPC dialogue?

### ISSUE THREAD

**Issue by DrfistMD** [EdenServer/community#1154]
Trying to complete Lure of the Wildcat in Windy, I've into an issue that Shantotto and Nanaa Mihgo won't give the correct Wildcat dialogue.  Nanaa I currently have an active quest for (Rock Racketeer), albeit that quest doesn't seem currently doable, and I've already opened a bug report around that.  So that's unfortunately somewhat understandable, albeit frustrating.

Shantotto on the other hand, I have completed every quest possible for her but yet she still refuses to give me the badge dialogue, and just loops the normal dialogue she gives you for completing all of her quests, blocking me from quest completion.

![shantotto](https://user-images.githubusercontent.com/46073012/53607977-fbf45780-3b8e-11e9-8c32-babfef538a64.PNG)

**Comment by DrfistMD** [comment]
Just wanted to update that I was able to complete Rock Racketeer, but am still unable to get the badge dialogue from Nanaa, she just loops the dialogue from completing that quest line, like Shantotto's behavior.

**Comment by DrfistMD** [comment]
So every 2 weeks or so I'd do a full circuit of the Lure of the Wildcat Windy NPCs to see if by chance something would change.  This past week I did not see any of the Lure of the Wildcat Windy quest dialogues, but when I talked to Ibwam this week he finally gave me the quest completion.  I don't recall completing any additional quests in Windy recently, so I unfortunately can't give any pointers as to what might have changed..  But as of this past week I was able to complete the quest.
